CMakeLists.txt

```diff
@@ -29,7 +29,8 @@ set(CATKIN_PKGS ${CATKIN_PKGS}
   time_series
 )
 find_package(catkin REQUIRED COMPONENTS ${CATKIN_PKGS})
-find_package(OpenCV 3 REQUIRED)
+find_package(OpenCV 3)
+find_package(Pylon REQUIRED)
```

Comment:
Pylon should be an optional dependency, otherwise everyone needs to install it.

If it is not found, the variable `${PYLON_FOUND}` will not be set (or set to false).  Add the following lines to be able to access this variable in the C++ code:

    if ${PYLON_FOUND}
        add_definitions(-DPYLON_FOUND)
    endif()

Then wrap all usages of Pylon in C++ with

    #ifdef PYLON_FOUND
       ...
    #endif

I think here this would only be the include and the python-binding of the pylon driver in `py_camera_types.cpp`

Reply:
Fixed this, thanks